src/codegen/llvm/codegen_llvm.cc

```diff
@@ -688,6 +688,9 @@ llvm::Value* CodeGenLLVM::CreateIntrinsic(const Call* op) {
       indices.push_back(i);
     }
     return builder_->CreateShuffleVector(v0, v1, indices);
+  } else if (op->is_intrinsic("bitcast")) {
```

Comment:
Can use reinterpret

https://github.com/dmlc/tvm/blob/ed984533008ad1cd5c6f6efccde71a3463b06514/src/codegen/llvm/codegen_llvm.cc#L671

Reply:
Nice! Din't notice that it was there. Thanks!